opensearch-dashboards/Dockerfile

```diff
@@ -0,0 +1,168 @@
+    --from=security-dashboards-plugin \
+    /stackable/src/opensearch-dashboards/security-dashboards-plugin/patchable-work/worktree/${OPENSEARCH_DASHBOARDS_SECURITY_DASHBOARDS_PLUGIN_VERSION} \
+    /stackable/security-dashboards-plugin
+
+# tar, xz - extract nodejs bundle
+# unzip - extract plugin from archive
+RUN <<EOF
+ARCH="${TARGETARCH/amd64/x64}"
+microdnf update
+microdnf install tar xz unzip
+microdnf clean all
+rm -rf /var/cache/yum
+curl "https://nodejs.org/dist/v${NODEJS_VERSION}/node-v${NODEJS_VERSION}-linux-${ARCH}.tar.xz" | tar -xJC /usr/local --strip-components=1
+npm install -g yarn@${YARN_VERSION} @cyclonedx/cdxgen@${CDXGEN_VERSION}
+EOF
+
+USER ${STACKABLE_USER_UID}
+
+RUN <<EOF
+ARCH="${TARGETARCH/amd64/x64}"
+cd "$(/stackable/patchable --images-repo-root=src checkout opensearch-dashboards ${PRODUCT_VERSION})"
+cp -r /stackable/security-dashboards-plugin plugins/security-dashboards-plugin
+NEW_VERSION="${PRODUCT_VERSION}-stackable${RELEASE_VERSION}"
+# Create snapshot of the source code including custom patches
+tar -czf /stackable/opensearch-dashboards-${NEW_VERSION}-src.tar.gz .
+HUSKY=0 yarn osd bootstrap --single-version=loose
+yarn build-platform --release
+mkdir /stackable/opensearch-dashboards
+tar -xzf target/opensearch-dashboards-${PRODUCT_VERSION}-linux-${ARCH}.tar.gz -C /stackable/opensearch-dashboards --strip-components=1
+cdxgen -t yarn --json-pretty
+cd plugins/security-dashboards-plugin
+yarn build
+cdxgen -t yarn --json-pretty
+unzip build/security-dashboards-${OPENSEARCH_DASHBOARDS_SECURITY_DASHBOARDS_PLUGIN_VERSION}.zip -d /stackable/security-dashboards-plugin
+mv /stackable/security-dashboards-plugin/opensearch-dashboards/securityDashboards /stackable/opensearch-dashboards/plugins/securityDashboards
+EOF
+
+
+RUN <<EOF
+# Change the group permissions already in the builder image to reduce
+# the size of the final image.
+# see https://github.com/stackabletech/docker-images/issues/961
+chmod -R g=u /stackable
+EOF
+
+FROM local-image/vector AS final
+
+ARG PRODUCT_VERSION
+ARG RELEASE_VERSION
+ARG NODEJS
+ARG STACKABLE_USER_UID
+
+ARG NAME="OpenSearch Dashboards"
+ARG DESCRIPTION="This image is currently not deployed by a Stackable Operator."
+ARG HOME=/stackable
+ARG OPENSEARCH_DASHBOARDS_HOME=${HOME}/opensearch-dashboards
+
+LABEL \
+  name="OpenSearch Dashboards" \
+  maintainer="info@stackable.tech" \
+  vendor="Stackable GmbH" \
+  version="${PRODUCT_VERSION}" \
+  release="${RELEASE_VERSION}" \
+  summary="The Stackable image for OpenSearch Dashboards." \
+  description="This image is currently not deployed by a Stackable Operator."
+
+# # https://github.com/opencontainers/image-spec/blob/036563a4a268d7c08b51a08f05a02a0fe74c7268/annotations.md#annotations
+LABEL org.opencontainers.image.version="${PRODUCT_VERSION}"
+LABEL org.opencontainers.image.revision="${RELEASE_VERSION}"
+LABEL org.opencontainers.image.title="${NAME}"
+LABEL org.opencontainers.image.description="${DESCRIPTION}"
+
```

Comment:
Labels for OpenShift could also be added, see e.g. https://github.com/stackabletech/docker-images/blob/25.7.0/hbase/Dockerfile#L93-L97.